gatsby-config.js

```diff
@@ -49,6 +49,13 @@ const plugins = [
       path: path.join(__dirname, 'content')
     }
   },
+  {
+    resolve: 'gatsby-source-filesystem',
+    options: {
+      name: 'images',
+      path: path.join(__dirname, 'static', 'img')
+    }
+  },
```

Comment:
Glad we're using `gatsby-plugin-image` for our docs! The images look kind of blurry. If I remember correctly from us fixing this with dvc.org, we need to add `quality: 90` to our `gatsby-remark-images` options.

Reply:
I remember the same solution to this problem, but after implementing it I don't notice any improvement. :thinking: